dss/events/handlers/index.py

```diff
@@ -76,44 +106,45 @@ def create_index_data(s3, bucket_name, manifest, logger):
             # of the query spec, so go with that, at least for now. Do so by substituting
             # dot for underscore in the key filename portion of the index.
             # As due diligence, additional investigation should be performed.
-            index_filename = file_info["name"].replace(".", "_")
+            index_filename = file_info[BundleFileMetadata.NAME].replace(".", "_")
```

Comment:
Yeah.  That's exactly what I mean.

One way to solve this is to make something an explicit escape character.  Then you can map things forwards and backwards authoritatively.